### DISCUSSION TITLE
❓ [Question] module 'trtorch' has no attribute 'Input'

### DISCUSSION THREAD

**Discussion by gaosanyuan** [pytorch/TensorRT#598] [Q&A] [answered]
Why the installed trtorch has no attribute 'Input'? Thanks
trtorch version: 0.3.0

**Comment by narendasan** [comment, marked as answer]
Input is added in TRTorch v0.4.0. In TRTorch v0.3.0 you pass in the shapes as tuples or dicts https://nvidia.github.io/TRTorch/v0.3.0/tutorials/getting_started.html#compiling-with-trtorch-in-python

**Reply by narendasan** [reply]
(moving to discussions as it seems to be mostly a usage question)
